jerry-core/ecma/operations/ecma-conversion.c

```diff
@@ -94,12 +94,21 @@ ecma_op_same_value (ecma_value_t x, /**< ecma value */
   const bool is_y_string = ecma_is_value_string (y);
   const bool is_y_object = ecma_is_value_object (y);
 
+#if ENABLED (JERRY_ES2015)
+  const bool is_x_symbol = ecma_is_value_symbol (x);
+  const bool is_y_symbol = ecma_is_value_symbol (y);
+#endif /* ENABLED (JERRY_ES2015) */
+
   const bool is_types_equal = ((is_x_undefined && is_y_undefined)
                                || (is_x_null && is_y_null)
                                || (is_x_boolean && is_y_boolean)
                                || (is_x_number && is_y_number)
                                || (is_x_string && is_y_string)
-                               || (is_x_object && is_y_object));
+                               || (is_x_object && is_y_object)
+#if ENABLED (JERRY_ES2015)
```

Comment:
Cannot we just simple insert the #ifdef -ed condition above the `|| (is_x_object && is_y_object));` line? Therefore no need to put the closing `);` into a new line.

Reply:
Thanks for the review. Updated.